src/parse.fs

```diff
@@ -50,7 +50,7 @@ module private ParseImpl =
     let number =
       let r = @"(\d+\.?\d*|\.\d+)([eE][-+]?[0-9]+)?"
       let conv s =
```

Comment:
I'd rather have the type annotation on the function declaration (`let conv (s: string) =`) instead of the next line.